scripts/monthly-server-report.py

```
@@ -0,0 +1,192 @@
+            ],
+            StartTime=start_dt,
+            EndTime=end_dt,
+            Period=86400,
+            Statistics=['Sum']
+        )
+        try:
+            if response['Datapoints']:
+                return int(sum([dp['Sum'] for dp in response['Datapoints']]))
+
+            print(f"⚠️ No datapoints for WAF metric: {metric_name}")
+            return 0
```

Ruff (0.14.6)

Consider moving this statement to an `else` block

(TRY300)